cmd/manager/main.go

```diff
@@ -17,157 +17,98 @@ limitations under the License.
 package main
 
 import (
-	"context"
 	"flag"
 	"os"
-	"runtime"
-	"time"
-
-	"github.com/openebs/node-disk-manager/pkg/apis"
-	"github.com/openebs/node-disk-manager/pkg/controller"
-	"github.com/openebs/node-disk-manager/pkg/env"
-	ndmlogger "github.com/openebs/node-disk-manager/pkg/logs"
-	"github.com/openebs/node-disk-manager/pkg/setup"
-	"github.com/openebs/node-disk-manager/pkg/upgrade"
-	"github.com/openebs/node-disk-manager/pkg/upgrade/v040_041"
-	"github.com/openebs/node-disk-manager/pkg/upgrade/v041_042"
-	"github.com/openebs/node-disk-manager/pkg/version"
-
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/ready"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"k8s.io/klog"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
Why is this import required?

Reply:
I think this can be used for out-of-cluster authentication . So we can develop the package locally, just need to point it to a K8s cluster. Although, the disks detected, etc would be on local machine.

Reply:
But as far as I see the code, its not used. So it can be removed.

Reply:
Keep only the most basic things so that it works as previously and further changes can be added later.